CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Redo OpenID Connect configuration [#2020](https://github.com/juanfont/headscale/pull/2020)
   - `strip_email_domain` has been removed, domain is _always_ part of the username for OIDC.
```

Comment:
Ok, I will need to think and research this a bit more. The main goal is that I want to go towards using "email style" for users in the Policy, so either the email as provided, or maybe just the "preferred_username" with `@` added to the end.

Reply:
I wouldn't modify the `preferred_username` in any way – it's intended to be used as-is.

As for policy preferences, Kanidm defaults to using an SPN that looks like an email address (eg: `user@kanidm.example.com`), but is probably not an actual email address. This can be changed on an application configuration level with `kanidm system oauth2 prefer-short-username $client_id`, which makes Kanidm return the `Person`'s `name` field as a `preferred_username` (eg: `user`) [^1].

[^1]: Kanidm's `name` field is similar to a "username" field. There are also `displayname` and `legalname` fields for longer forms of the above.